server/server.go

```diff
@@ -327,9 +326,6 @@ func NewServer(cfg *config.Config, driver IDriver) (*Server, error) {
 		}
 	}
 
-	// Init rand seed for randomBuf()
-	rand.Seed(time.Now().UTC().UnixNano())
-
```

Comment:
What?

Reply:
https://pkg.go.dev/math/rand#Seed

> If Seed is not called, the generator is seeded randomly at program startup.

Not sure do we need it anymore.